In [3]:
import pandas as pd
import numpy as np
import seaborn as sns

In [4]:
url = "https://raw.githubusercontent.com/guyuliel/Data-Science/main/Heart_Attack_Data_Set.csv"

In [5]:
myfile = pd.read_csv(url)
myfile

age  sex  cp  trestbps  chol  fbs  restecg  thalach  exang  oldpeak  \
0     63    1   3       145   233    1        0      150      0      2.3   
1     37    1   2       130   250    0        1      187      0      3.5   
2     41    0   1       130   204    0        0      172      0      1.4   
3     56    1   1       120   236    0        1      178      0      0.8   
4     57    0   0       120   354    0        1      163      1      0.6   
..   ...  ...  ..       ...   ...  ...      ...      ...    ...      ...   
298   57    0   0       140   241    0        1      123      1      0.2   
299   45    1   3       110   264    0        1      132      0      1.2   
300   68    1   0       144   193    1        1      141      0      3.4   
301   57    1   0       130   131    0        1      115      1      1.2   
302   57    0   1       130   236    0        0      174      0      0.0   

     slope  ca  thal  target  
0        0   0     1       1  
1        0   0     2       1  
2        2   0     2       1  
3        2   0     2       1  
4        2   0     2       1  
..     ...  ..   ...     ...  
298      1   0     3       0  
299      1   0     3       0  
300      1   2     3       0  
301      1   1     3       0  
302      1   1     2       0  

[303 rows x 14 columns]

###### cp - דלקות עבר בקרום הלב 
###### trestbps - לחץ דם במנוחה
###### chol - כולסטרול 
###### fbs - רמת סוכר גבוה בדם (מעל 120)
###### restecg - בדיקת אקג (במנוחה)
###### thalach - דופק מקסימלי 
###### exang - ####תעוקת חזה 
###### oldpeak - ##### תעוקת חזה 
###### slope - השיפוע של ST
###### ca - בחינה בעזרת בדיקה האם יש הפרעות בזרימות הדם מהלב ואל הלב
###### thal - ####אנמיה
###### target - סיכוי על הנייר להתקף לב 0 - פחות סיכוי 